notebooks/depth_time_series.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
<p>what are the shaded regions?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Those shaped regions are because there were a range of values for the corresponding year, and <code>seaborn</code> took the median value for plotting the line. These regions help us identify the <code>minimal</code> and <code>maximal</code> depth for a species.</p>